Channels/UserInterface/02a_Offers.md

```diff
@@ -0,0 +1,877 @@
+
+- ![Columns](../../Assets/Icons/Columns.png "[Columns]") Columns (x)   
+  Click this button to display the columns bar and customize the displayed columns and the order of columns in the list. The *x* indicates the number of columns that are currently displayed in the list.
+
+- ![Filter](../../Assets/Icons/Filter.png "[Filter]") Filter (x)   
+  Click this button to display the filter bar and customize the active filters. The *x* indicates the number of filters that are currently active.
+
+- [x]     
+  Select the checkbox to display the editing toolbar. If you click the checkbox in the header, all offers in the list are selected.
+
+- *Change status to*  
+  Click the drop-down list to change the offer status. The selected status is displayed in the *Pending status/Errors* column. This drop-down list is only displayed if the checkbox of at least one offer is selected. The following options are available:   
+  - **Active**   
+    The offer is active. It is displayed in the marketplace, where it can be sold.   
+  - **Inactive**   
+    The offer is inactive. It is not longer displayed in the marketplace and thus cannot be sold anymore.   
+  - **Not available**   
+    The offer is deleted from the marketplace.   	
+
+- ![Edit](../../Assets/Icons/Edit03.png "[Edit]") (Edit)           
+  Click this button to edit the selected offer from the list. This button is only displayed if a single checkbox in the list of offers is selected. Alternatively, you can click directly a row in the list to edit the corresponding offer.  
+  The *Edit offer "Offer SKU"* view is displayed, see [Edit offer](#edit-offer).
+
+
+The list displays all available offers. Depending on the settings, the displayed columns may vary. All fields are read-only. 
+
+- *SKU*   
+  Stock Keeping Unit. Identification number of the product to the offer. 
+ 
+- *Variant status*  
+  Variant type of the offer. The following options are available:  
+  - **Master offer**   
+    This offer refers to a master product.
+  - **Variant offer**   
+    This offer refers to a variant.
+  - **Standalone Offer**  
+    This offer refers to a single product.
+
+- *Status*  
+  Status of the offer. The following options are available:  
+  - **Active**   
+    The offer is active. It is displayed in the marketplace, where it can be sold.   
+  - **Inactive**   
+    The offer is inactive. It is not longer displayed in the marketplace and thus cannot be sold anymore.   
+  - **Not available**   
+    The offer is deleted from the marketplace.   
+    The *Change offer status* confirmation window is displayed. If the variant status of the selected offer equals **Master offer**, an additional toggle is displayed within the confirmation window.   
```

Comment:
"within the confirmation window"? Bedeutet das ein "Change offer status" Fenster mit üblicher "Are you sure...?" Frage + toggle zusäztlich, wenn offer ist ein Master offer? 
Im Screenshot sieht es so wie zwei unterschiedliche Fenster aus? Sind sie zwei Fenster, die nacheinander/nebeneinander vorkommen (können, je nach offer variant)?